pkg/test/resourcefixture/testdata/basic/backupdr/v1beta1/backupdrbackupvault/backupdrbackupvault-labels/_generated_object_backupdrbackupvault-labels.golden.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: backupdr.cnrm.cloud.google.com/v1beta1
```

Comment:
So this PR is doing a lot of things at once.  Can we prompt alpha -> beta in one PR, can we add label support in another, and can we add this test in a third.  I suggest the order should be 

1) Remove spec.labels
2) Promote alpha -> beta
3) Add Label test showing that it doesn't work
4) Fix labels (so test change is just label support working)